### PR TITLE
Modify local-pod-under-test pod definition with replica eq 2 and anti…

### PR DESCRIPTION
**Pull request by wying3**
…Affinity policy

### CODE REVIEW

**Review by HamadiSE — COMMENTED** [review]
.

File: local-test-infra/local-pod-under-test.yaml [hunk 1]
```diff
@@ -28,3 +28,14 @@ spec:
             limits:
               memory: 512Mi
               cpu: 0.25
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: test
```
Comment (jun-rh) [inline local-test-infra/local-pod-under-test.yaml:36]: We won't merge this PR, but shouldn't the key be "app"?

File: local-test-infra/local-pod-under-test.yaml [hunk 2]
```diff
@@ -28,3 +28,14 @@ spec:
...
+                operator: In
+                values:
+                - test
+            topologyKey: "kubernetes.io/hostname"
```
Comment (jun-rh) [inline local-test-infra/local-pod-under-test.yaml:40]: should this be hostname or node?

Comment (wying3) [reply local-test-infra/local-pod-under-test.yaml:40]: from my understanding, key can be any term, e.g, Zone, hostname, node, rack or power

**Pull request closed without merging**